dev-data/courses.data.ts

```diff
@@ -196,4 +197,25 @@ export const courses: Course[] = [
       accentColor: '#7356BF',
     },
   },
+  {
+    id: '10',
+    title: COURSE_TITLES.SHORT_TRACK,
+    subTitle: null,
+    descriptionUrl: COURSE_LINKS.SHORT_TRACK,
+    altTitle: COURSE_TITLES.SHORT_TRACK,
+    iconSrc: reactAngular,
```

Comment:
Icon WO border. I remember we would like to remove all borders there, but I suppose we should remove them at the same time

Reply:
update icons